File: 3753-maximum-difference-between-even-and-odd-frequency-i/3753-maximum-difference-between-even-and-odd-frequency-i.js
/**
 * @param {string} s
 * @return {number}
 */
var maxDifference = function(s) {
    var count = {};
    var minEven = s.length;
    var maxOdd = 0;
    for(const c of s) count[c] = (count[c] || 0) + 1;
    for(var i of Object.values(count)){
        if(i == 0) continue;
        if( i % 2 == 1 ) maxOdd = Math.max(maxOdd, i);
        else minEven = Math.min(minEven, i);
    }
    return maxOdd - minEven;
};

File: 3753-maximum-difference-between-even-and-odd-frequency-i/3753-maximum-difference-between-even-and-odd-frequency-i.cs
public class Solution {
    public int MaxDifference(string s) {
        var count = new int[26];
        foreach(char c in s){
            count[c - 'a']++;
        }
        int maxOdd=0;
        int minEven = s.Length;
        foreach(var i in count){
            if(i == 0) continue;
            if(i % 2 == 1) maxOdd = Math.Max(maxOdd, i);
            else minEven = Math.Min(minEven, i);
        }
        return maxOdd - minEven;
    }
}